languages/elixir/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -0,0 +1,21 @@
+### General
+
+- The [atom type is described here][atom]
+
+### 1. Return the logging code label
+
+- You can use a [`cond/1` function][cond] to elegantly handle the various log codes.
```

Comment:
This line gives away the solution. Maybe you could rephrase a bit to not include the solution?